src/xenia/kernel/xam/xam_info.cc

```diff
@@ -58,6 +58,9 @@ typedef enum _MODE { KernelMode, UserMode, MaximumMode } MODE;
 dword_result_t XamFeatureEnabled_entry(dword_t unk) { return 0; }
 DECLARE_XAM_EXPORT1(XamFeatureEnabled, kNone, kStub);
 
+dword_result_t XamGetStagingMode_entry() { return 1; }
```

Comment:
Shouldn't it return 0 since 1 is staging/debug mode and 0 is production?

Reply:
Added `staging_mode` config variable to General category. However, Kernel or XConfig category might be more suitable?

Reply:
Probably Kernel imo